### ISSUE TITLE
Problem with   rake 'single_events:generate'

### ISSUE THREAD

**Issue by bitboxer** [hacken-in/website#539]
Currently we have a problem with the events. Sometimes the evens are recreated and old ones removed. I think the problem is causes by `rake 'single_events:generate'`.

You can see this by simply opening a calender page, wait a day and then try to click on the items. Some of them will be missing because they have been recreated :sob:

@moonglum do you have an idea why this might happen?

**Comment by moonglum** [comment]
Oh no :frowning: Currently no idea, would need to investigate!

**Comment by bitboxer** [comment]
I have a suspicion that this is _again_ an IceCube milliseconds issue :sob: . I tried to pinpoint it, but without any success. @moonglum I would love to pair with you to fix this.

**Comment by moonglum** [comment]
Yep, let's do that! Damn this stupid IceCube milliseconds.... :unamused:
